installer/PowerToysSetup/PowerToysInstaller.wixproj

```diff
@@ -31,32 +30,6 @@ call powershell.exe -NonInteractive -executionpolicy Unrestricted -File $(MSBuil
   </PropertyGroup>
   <PropertyGroup>
     <RunPostBuildEvent>Always</RunPostBuildEvent>
-    <PostBuildEvent>

```

Comment:
Is this deleted by mistake? I don't see a change that replaces this logic

Reply:
Yes I must have done that by mistake. Thank youi